### PR TITLE
[13.0][ADD] sale_coupon_partner: New module that links a sale_coupon_program with a partner

### PR DESCRIPTION
**Pull request by CarlosRoca13**
This module adds the possibility of link a sale.coupon.program with a res.partner and allow to filter it on Sale Analysis.

cc @Tecnativa TT30848

Please @pedrobaeza @ernestotejeda review this.

### CODE REVIEW

File: sale_coupon_partner/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+=======================
+Sale Coupon Mas Mailing
```
Comment (pedrobaeza) [inline sale_coupon_partner/README.rst]: Incorrect manifest name

File: sale_coupon_partner/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Coupon Mas Mailing",
```
Comment (pedrobaeza) [inline sale_coupon_partner/__manifest__.py]:
```suggestion
    "name": "Sale Coupon Partner",
```

**Comment by CarlosRoca13** [comment]
@pedrobaeza Changed

**Review by pedrobaeza — APPROVED** [review]

**Review by ernestotejeda — COMMENTED** [review]
Code review: :+1:

**Pull request merged**